YvesIraguha/lost-and-found-api
# #171923516 setup testing environment

#### What does this PR do?
* Setting testing environment
#### Description of Task to be completed?
* Setting *jest* for testing
* Installing *supertest* for request testing
#### How should this be manually tested?
* Clone the repo and checkout `ch-setup-testing-environment-#171923516`
* run `npm install` to install dependencies
* fill `.env` with correct credentials
* Set your environment to testing by running
```javascript
//For MacOs & Git Bash
$export NODE_ENV="testing"  
```
```javascript
//For windiw Powershell
$env:NODE_ENV="testing"
```
```javascript
//For Window cmd
set NODE_ENV=testing
```
* run `npm test`
> you should see a pass of 3 test
#### Any background context you want to provide?
* If you want to make a test fail, you may change any of the test suit in /tests/controller.test.js
#### What are the relevant pivotal tracker stories?
#171923516
